internal/pkg/task/task.go

```diff
@@ -55,10 +56,13 @@ func taskFamilyName(groupName string) string {
 }
 
 func newTaskFromECS(ecsTask *ecs.Task) *Task {
+	eni, _ := ecsTask.ENI() // tasks created by `task run` are fargate tasks that come with ENI by default.
```

Comment:
Hmm i think we should still handle with the possible error here, since it is safer and doesn't hurt to handle them.

Reply:
Updated!